### ISSUE TITLE
Cannot override default jasmine timeout when using grunt runner

### ISSUE THREAD

**Issue by christinedraper** [jasmine-contrib/grunt-jasmine-node#57]
In my spec helper, I have changed the default timeout to 10 seconds using:

`jasmine.getEnv().defaultTimeoutInterval = 10 *1000`

This works fine when run through the jasmine-node timeout. However, when I use the grunt plugin, it goes back to the standard 5 seconds default.

I am using node 0.10.29, grunt 0.4.5, jasmine-node 1.14.3, grunt-jasmine-node 0.2.1

**Comment by marcincichon** [comment]
Same issue here. @christinedraper have you managed to resolve it?

**Comment by christinedraper** [comment]
No. I resorted to putting a longer timeout on specific tests, i.e.  

	it("should do something that takes a while", function(done)
	{
		request.get(.... }, function(error, response, body)
		{
			...
			done();
		});
	}, request.longerTimeout);

Not ideal though.
